numpy/random/_mtrand/distributions.c

```diff
@@ -447,6 +447,10 @@ long rk_binomial(rk_state *state, long n, double p)
 {
     double q;
 
+    if ((n == 0LL) || (p == 0.0f)) {
```

Comment:
If this file is going to stick around, you should copy and paste from the version in master to get all of the changes.

Reply:
done, and added back the functions that moved to distributions.h

Reply:
This is the numpy copy fo the file.  It shoudl be untouched and copied from NumPy master, not randomgen.  Sorry fi I wasn't clean.  It shoudl have `rk_` functions.

Reply:
Did you push your last set of changes?  This file appears to be the same in the github view.